examples/csj/s0/conf/train_conformer.yaml

```diff
@@ -0,0 +1,80 @@
+# network architecture
```

Comment:
we can only checkin the basic training yaml(train_conformer.yaml) you used in this PR, and add other configures when you add new result  in the future.

so we can keep small PR which is perferred for both review and development

Reply:
sure, only one yaml is left now